pandas/io/tests/test_parsers.py

```diff
@@ -2665,7 +2660,174 @@ def test_empty_header_read(count):
             test_empty_header_read(count)
 
 
-class TestPythonParser(ParserTests, tm.TestCase):
+class CompressionTests(object):
+    def test_zip(self):
+        try:
+            import zipfile
+        except ImportError:
+            raise nose.SkipTest('need zipfile to run')
+
+        with open(self.csv1, 'rb') as data_file:
+            data = data_file.read()
+            expected = self.read_csv(self.csv1)
+
+            file_name = 'test_file.zip'
+            with tm.ensure_clean(file_name) as path:
+                tmp = zipfile.ZipFile(path, mode='w')
```

Comment:
@jreback thanks for clarifying. I was using the previous convention. I'll clean this up now.